Core/GameEngine/Source/GameNetwork/LANGameInfo.cpp

```diff
@@ -92,6 +92,7 @@ LANGameInfo::LANGameInfo()
 	//Initializtions missing and needed
 	m_lastHeard = 0;
 	m_next = NULL;
+	m_isDirectConnect = false;
```

Comment:
Are you certain this caused uninitialized memory use on runtime? LANGameInfo is allocated through Game Memory overloads, which means its memory is memset 0 before the constructor is called. So if you look at this object in debugger, then this field should 0 regardless.

Reply:
No, I think you're right here. Is it still good practice to do it explicitly here? If not, I can just drop that commit.